README.md

```diff
@@ -14,6 +14,20 @@ Provided rules:
 To test a `pybind_extension`, the most common approach is to write the test in
 python and use the standard `py_test` build rule.
 
+Provided targets:
```

Comment:
Can you reformat the text to align with the rest of the file? In particular, adhering to the 80-column limit.

This helps keep it readable in its raw form.